DESCRIPTION

```diff
@@ -36,57 +36,56 @@ Imports:
     methods,
     S4Vectors,
     crayon,
-    Matrix
+    Matrix,
+    pkgconfig,
+    AnnotationDbi,
+    DESeq2,
+    Rtsne,
+    Seurat,
+    betareg,
+    boot,
+    broom, 
+    class, 
+    clusterProfiler, 
+    e1071, 
+    edgeR,
+    functional, 
+    glmmSeq, 
+    glmmTMB, 
+    limma, 
+    lme4, 
+    matrixStats,
+    msigdbr, 
+    org.Hs.eg.db,
+    org.Mm.eg.db, 
+    pbapply, 
+    pbmcapply,
+    survival,
+    survminer, 
```

Comment:
@chilampoon and @william-hutchison we need to drop all these new `Imports`. If Bioconductor does not accept on-the-fly installation anymore, we rather print the message to install if the packages are not installed, and then use 

```
<package>::<function>
```

but without adding those dependendies to `@importFrom`, not in DESCRIPTION as Imports

Reply:
yead I agree on too many Imports, however directly add `<package>::<function>` inside a function without specifying `@importFrom' will trigger a warning saying '::' or ':::' imports not declared from: xyz packages...